Графика для всех страниц

Alex88GitHub opened the pull request:
---
:mortar_board: [Графика для всех страниц](https://up.htmlacademy.ru/htmlcss/36/user/2254559/tasks/7)

:boom: https://htmlacademy-htmlcss.github.io/2254559-sedona-36/3/

Alex88GitHub committed:
Графика

добавляет папки и изображения

Alex88GitHub committed:
Update index.html

добавляет логотип Sedona

Alex88GitHub committed:
графика

Добавляет контентные изображения в разметку

Alex88GitHub committed:
Графика

Логотип академии = изображение-ссылка

Alex88GitHub committed:
графика

добавляет иконки

4elbik commented on index.html:
```diff
@@ -42,8 +42,8 @@
       <h2 class="vissually-hiden">Добро пожаловать!</h2>
-      <img class="main-page-photo" src="#" alt="Фотография гор">
-      <img class="welcome-img" src="#" alt="Welcome to the gorgeous Sedona. Because the Grand Canyon sucks!">
+      <img class="main-page-photo" src="images/content/index-background.jpg" width="1200" height="485" alt="Фотография гор">
```
Точно это у нас контентное изображение?

Alex88GitHub replied:
Да произошла небольшая путаница. В лайве по разметке говорилось, что изображение в херо блоке связано логически с текстом, поэтому является контентным изображением. Однако в лайве по графике все же сказали, что это фоновое изображение, следовательно декоративное. Сейчас я из кода убрал разметку. Вставлю его потом через CSS. По точкам и названиям файлов замечания устранил.

4elbik reviewed (commented):
1. В `alt` у нас после описания должна стоять точка (`.`)
2. Названия файлов должно быть без заглавных букв [PROJ-03](https://up.htmlacademy.ru/htmlcss/36/criteries#PROJ-03)
___
В остальном - идеально. Графика нарезана отлично, форматы выбраны те, что нужны)

Alex88GitHub committed:
графика для всех страниц

Добавляет точки в Alt

Alex88GitHub committed:
графика для всех страниц

1. корректирует название файлов
2. меняет директорию расположения изображений
3. убирает из разметки код фонового изображения hero блока.

Alex88GitHub committed:
графика для всех страниц

Убирает заглавную букву в  названии файла "logo-sedona". Точка в alt "логотип html академия.!"
